populate_dev_data_handler.go

```diff
@@ -68,6 +70,33 @@ func populateDevData(w http.ResponseWriter, r *http.Request) {
         },
     }
 
+    // Get the redirects, but don't follow them.
+    var errUseLastResponse = errors.New("hey... don't redirect")
+    client := urlfetch.Client(ctx)
+    client.CheckRedirect = func(req *http.Request, via []*http.Request) error {
+        return errUseLastResponse
+    }
+
+    for _, platform := range([]string {
```

Comment:
Tiny nit: can `platform` be `browserName`?

Reply:
Done.